packages/api-specification/api/application.ts

```diff
@@ -0,0 +1,33 @@
+export interface ApplicationAPI {

```

Comment:
We might need to clarify the meaning of `Application` for OpenFin, as currently OpenFin windows that are not child windows are created using `fin.desktop.Application()`. Would this API be used for each OpenFin application, or ContainerJS application? If we want this per ContainerJS application, then we might need to rethink the `setBadgeCount()` or the window creation implementation as each ContainerJS application could have multiple top level windows, which could have different badge counts.